### ISSUE TITLE
Added the mocks

### ISSUE THREAD

**Issue by ccoria** [NinjaReader/AndroidNinjaReader#7]
@alonrz and @mvinceable,

I Added the mocks to this project. I think perhaps we will need to create an action bar to make it look better.. I have also added the models, icons and updated the styles..

I will work on the stats page tomorrow.

Just to make sure we are in the same page:
- Vince will bind the mocks to his backend
- Alon will work on the intents to add articles and eventually on alerts
- Cassiano will do the stats page

Is that right? Am I missing something? Do you guys agree?

There is this action bar styling that we may need to do. I think whoever is free first could take a look on that.

Thanks,
Cassiano

**Comment by mvinceable** [comment]
That sounds good to me.

Thanks,
Vince

**Comment by mvinceable** [comment]
Hey Guys,

There were some bugs with parse login that I fixed now.  We should only see
the reading list when there's a proper current user.  I also added a
ParseObject for Article.  Please pull for the latest as I've also
restructured the files for better organization.

Thanks,
Vince

**Comment by mvinceable** [comment]
Hi,

In the ReadingListActivity I added an example for getting article info from
Readability and storing values we care about into Parse.  I'm seeing live
data in Parse now.

Thanks,
Vince

**Comment by mvinceable** [comment]
Please pull the latest changes before working further as I had to resolve some naming conflicts.

**Comment by alonrz** [comment]
Hi guys, great work.
I will add the action bar too as I need to add a button to it anyways.

**Comment by alonrz** [comment]
Hey Vince, is the login big you fixed related to what I sent you?

**Comment by ccoria** [comment]
Great additions, thanks guys.

I also added the fragment where we can load an article to be read (ArticleFragment.java). It is there but it is not being loaded.

Hey @mvinceable, are you planning to remove this example from our namespace? I can do it but the chance of having conflicts is very high..

On the stats, I researched a little and I will user this library to the graphs: https://github.com/HackPlan/AndroidCharts

Would you guys be able to meet tomorrow from 2PM to 3PM to talk about the last details?

Thanks,
Cassiano

**Comment by alonrz** [comment]
im free. can you create an invite in yahoo cal?

**Comment by ccoria** [comment]
Sure, done

**Comment by mvinceable** [comment]
Guys, anyone should be free to make any changes to this repo.

@ccoria I didn't do anything with ArticleFragment.  Perhaps if it contained live data it might be helpful.

**Comment by alonrz** [comment]
I wonder if we should invest time with real data at this point, or just static screens. When I try, there are all these bugs that float up. So I didnt do the saving part properly yet. Just to demo there is the share and add article dialog from the app. I set the structure for future dev but when I tried there were a bunch of crashes and scenarios that needs attention. What do you guys think?

**Comment by ccoria** [comment]
Guys, 

I created this profile/stats page. What do you think?

I still have to integrate with this project.. Sorry the Graphs API was kind of hard to customize. And I had to change to the hellocharts because the other one was not being nice with me.

I think that we have just to tell the story.. if we have it working, ok, but if not, lets try to tell a good story.. =)

Thanks,
Cassiano
![screen shot 2015-02-26 at 12 31 20 am](https://cloud.githubusercontent.com/assets/69435/6388454/e5494f26-bd4e-11e4-9e68-d657d0ae5139.png)

**Comment by ccoria** [comment]
@mvinceable I agree and I would change the namespace but since you were working on the classes itself I was concerned about conflicts. But sure, we are free do to whatever.. =)

**Comment by mvinceable** [comment]
With the latest changes I checked in, we have a working prototype that is integrated with Parse that can

1) List articles user has saved
2) Mark articles as read via long press
3) Show word count for all articles and lead image if available

I also went in and fixed all the bugs I found.  Should be pretty stable now.

@alonrz I was able to merge with your changes also
@ccoria that chart looks nice.  I think we can go even more granular and add word counts?
